nx.json

```diff
@@ -59,7 +68,65 @@
       "tags": ["scope:tooling"]
     },
     "components": {
-      "tags": ["scope:components"]
+      "tags": ["scope:components"],
+      "implicitDependencies": [
```

Comment:
As this is another list where we have to insert components when they are created, we need some way to handle it more easy. It will definitely happen that developers will forget to add it to one or more of these lists.
cc @ffriedl89 @tomheller

Reply:
Do we need a follow up on this one?

Reply:
What's the implication if someone forgets to add a library here? @lukasholzer

Reply:
No implications - this only affects the visual dep graph of nx